extensions/skyportal/static/js/components/TNSInfo.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+
+import * as tnsInfoActions from "../ducks/tnsInfo";
+
+const TNSInfo = ({ objID }) => {
+  const dispatch = useDispatch();
+  const [requestSubmitted, setRequestSubmitted] = useState(false);
+  const tnsInfo = useSelector((state) => state.tnsInfo);
+
+  useEffect(
+    function fetchTNSInfo() {
+      if (!requestSubmitted) {
+        dispatch(tnsInfoActions.fetchTNSInfo(objID));
+        setRequestSubmitted(true);
+      }
+    },
+    [objID, dispatch, requestSubmitted]
+  );
+
+  if (tnsInfo === null) {
+    return <>Fetching TNS data...</>;
+  }
+  return <span>{tnsInfo?.name ? tnsInfo.name : `No matches found`}</span>;
```

Comment:
This is very nice! I'd just turn it into a link with `target="_blank"` and `rel="noopener noreferrer"` if there's a match:
For example, if the name is "AT 2021mgf", the link will be
https://www.wis-tns.org/object/2021mgf
(i.e. split by whitespace and pick the last element.)

Reply:
Will do!

Reply:
Done